src/server/polaris-server.ts

```diff
@@ -104,6 +106,19 @@ export class PolarisServer {
         if (this.polarisServerConfig.plugins) {
             plugins.push(...this.polarisServerConfig.plugins);
         }
+
+        if (!this.polarisServerConfig.supportedRealities) {
+            this.polarisServerConfig.supportedRealities = new RealitiesHolder();
```

Comment:
this.polarisServerConfig.supportedRealities = this.polarisServerConfig.supportedRealities || new RealitiesHolder();